Automattic/wp-calypso
# Remove jetpack 2-yr products from checkout variant dropdown selector.

This is merely a temporary PR that removes the ability for users to see and therefore purchase 2 new Jetpack bi-yearly plans that are currently in process of getting created and setup. Once everything is setup (like Intro Offers, etc.) and working correctly with the bi-yearly products, then I will come back and remove this code, making the bi-yearly products visible and purchasable for users.

Related to # 1201210512993648-as-1204625396673339/f
See P2: pbNhbs-6CA-p2 

## Proposed Changes

- Removes products: `PRODUCT_JETPACK_BACKUP_T1_BI_YEARLY` and `PLAN_JETPACK_SECURITY_T1_BI_YEARLY` from the checkout variant dropdown selector. 

## Testing Instructions

- Apply diff D111218-code to your WPCOM sandbox. (if it is not already deployed)
- sandbox `public-api.wordpress.com` .
- Add `define( 'USE_STORE_SANDBOX', true );` to your `0-sandbox.php` file.
- Spin up this PR: `git fetch && git checkout update/remove-jetpack-2yr-variants && yarn start`
- Go to: http://calypso.localhost:3000/checkout/jetpack/jetpack_backup_t1_yearly
- Verify that the checkout page loads and that the billing term variant dropdown selector does **not** show a "2 year" option.
- go to: http://calypso.localhost:3000/checkout/jetpack/jetpack_security_t1_yearly
- Verify that the checkout page loads and that the billing term variant dropdown selector does **not** show a "2 year" option.




*

## Pre-merge Checklist
